kikit/fab/neodenyy1.py

```diff
@@ -0,0 +1,164 @@
+import click
+from pcbnewTransition import pcbnew
+import csv
+import os
+import sys
+import shutil
+import re
+from pathlib import Path
+from kikit.fab.common import *
+from kikit.common import *
+from kikit.export import gerberImpl
+mm_ius = 1000000.0
+footprintRe = [
+    (re.compile(r'Capacitor_SMD:C_(\d+)_.*'), 'C_{}'),
+    (re.compile(r'Diode_SMD:D_(\d+)_.*'), 'D_{}'),
+    (re.compile(r'Inductor_SMD:L_(\d+)_.*'), 'L_{}'),
+    (re.compile(r'Resistor_SMD:R_(\d+)_.*'), 'R_{}'),
+    (re.compile(r'Crystal:Crystal_SMD_(.*?)_.*'), 'CRYSTAL_{}')
+]
+
+def noFilter(footprint):
+    return True
+
+def collectBom(components, ordercodeFields, ignore):
```

Comment:
This function is copy & paste copy of `kikit.fab.jlcpcb.collectBom`, only check for the field `JLCPCB_IGNORE` is missing. I suggest generalizing this function to take vendor-specific filtering function, moving it into the `common` module and using it both here and in the JLC PCB assembly (not sure about PcbWay assembly at the moment).

Reply:
Moved

Reply:
I don't see this function moved in your most recent commits.

Reply:
I removed this function in neodenyy1.py and jlc.py and put it into common.